getting_started.rst

```diff
@@ -0,0 +1,43 @@
+
+Getting Started
+=====================
+
+First Steps
+###############
+
+- Join our chat_ and say hello in any of the rooms!
+
+- Find a feature ticket that you'd be exited to work on. There are two pieces of rovercode, and therefore two collections of tickets: rovercode-web_ (the web app) and rovercode_ (the code that runs on the rover hardware).
+
+- Get set up by following the READMEs: `rovercode-web README <https://github.com/rovercode/rovercode-web/blob/development/README.md>`_ or `rovercode README <https://github.com/rovercode/rovercode/blob/development/README.md>`_. We work hard to make setup work right, but remember to reach out in the chat_ room if you have any issues!
```

Comment:
Should these link to the `master` branch?

Reply:
I wasn't sure. I guess so? I was thinking that while we're under rapid development, it's best to look at the development branch? But if we deploy often, it should be ok.